src/main/webapp/definitions/alfresco-core.yaml

```diff
@@ -8472,6 +8472,49 @@ paths:
           description: Unexpected error
           schema:
             $ref: '#/definitions/Error'
+  '/nodes/{nodeId}/linked-ruleset':
+    post:
+      x-alfresco-since: "7.3.0"
+      tags:
+        - rules
+      summary: Link a ruleset to another folder
```

Comment:
Do we know how / where we can show:

- who we link to 
- who links to us 

Maybe that will part of GET /node/{nodeId}/rulesets/-default ? ( see https://github.com/Alfresco/rest-api-explorer/pull/161/files )